x/denommetadata/module.go

```diff
@@ -131,7 +140,7 @@ func (am AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
 // InitGenesis performs the module's genesis initialization.
 // Returns an empty ValidatorUpdate array.
 func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, gs json.RawMessage) []abci.ValidatorUpdate {
-
+	_ = am.evmKeeper.DeployVirtualFrontierBankContractForBankDenomMetadataRecord(ctx, am.evmKeeper.GetParams(ctx).EvmDenom)
```

Comment:
why it's part of the `denommetadata` module?

Reply:
I place it there so Tza can develop his part. Can remove when PR merge.

Reply:
@VictorTrustyDev can you please remove? @ItzhakBokris tested. seems good. 

possibly we'd wanna write init genesis for the vfc in case of starting from a genesis with multiple denom metadatas to deploy vfc for them. wdyt?

Reply:
@omritoptix problem is no account have balance to transfer

Reply:
FYI: removed

Reply:
@omritoptix @mtsitrin I have just added
[`x/denommetadata` triggers hook at genesis for created bank denom metadata](https://github.com/dymensionxyz/dymension/pull/668/commits/8a7c0e42ed67ccd739262f2bd1892da0b7731aac)
Ofc the filter still applied (`ibc/*` only), wdyt?